barretenberg/cpp/src/barretenberg/ultra_honk/decider_proving_key.hpp

```diff
@@ -52,7 +54,7 @@ template <IsHonkFlavor Flavor> class DeciderProvingKey_ {
         circuit.finalize_circuit(/*ensure_nonzero=*/true);
 
         // Set flag indicating whether the polynomials will be constructed with fixed block sizes for each gate type
-        const bool is_structured = (trace_structure != TraceStructure::NONE);
```

Comment:
Why would this const need to go away?

Reply:
just because it used to be a local variable and now its a member and cpp guidelines suggest no const class members. I've updated it to be effectively const by making it private and adding a getter